SumoLogic.Logging.NLog/BufferedSumoLogicTarget.cs

```diff
@@ -231,9 +234,8 @@ private SumoLogicMessageSender SumoLogicMessageSender
         public void ActivateConsoleLog()
         {
 #if netfull
-            this.LogLog = new ConsoleLog();
-#else
-            this.LogLog = new DummyLog();
+            if (!this.LogLog.IsConsoleEnabled)
+                this.LogLog = new InternalLoggerLog(GetType().Name + ": ", new ConsoleLog());

```

Comment:
If the NLog InternalLogger Console-output is already enabled, then it will cause double-logging if also adding `ConsoleLog`. I have now added a comment that explains, and also removed `netfull`

Reply:
Thanks for the explanation! make sense